apps/components-e2e/src/components/table/table-simple/table-simple.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component } from '@angular/core';
+
+const DATA = [
+  { name: 'I' },
+  { name: 'II' },
+  { name: 'III' },
+  { name: 'IV' },
+  { name: 'V' },
+];
+
+@Component({
+  selector: 'dt-e2e-table-simple',
+  templateUrl: 'table-simple.html',
+})
+export class DtE2ETableSimple {
+  disabled = false;
+
+  loading = false;
+
+  dataSource = DATA;
+
+  toggleTableData(): void {
+    if (this.dataSource.length === 0) {
+      this.dataSource = DATA;
+    } else {
+      this.dataSource = [];
+    }
```

Comment:
NIT: 
```suggestion
    this.dataSource = this.dataSource.length === 0 ? DATA : [];
```

Reply:
I approve of this NIT, but as this is in a test component i think I will leave it at that.